tests/sops.sh

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
```

Comment:
@prachirp FYI, this needs to be cleaned up https://github.com/GoogleContainerTools/kpt/issues/976

Reply:
Cleaning up depends on features in the next kpt release which I expect to happen sometime this week.